rotkehlchen/api/v1/resources.py

```diff
@@ -1353,6 +1364,10 @@ def put(self, async_query: bool, filepath: Path) -> Response:
         @require_loggedin_user()
         @use_kwargs(upload_schema, location='form_and_file')
         def patch(self, async_query: bool, filepath: FileStorage) -> Response:
+            _, tmpfilepath = tempfile.mkstemp()
```

Comment:
note my comment in the other import data endpoint. This logic shoul dnot be here